src/librustc_mir/util/borrowck_errors.rs

```diff
@@ -53,12 +53,12 @@ impl<'cx, 'tcx> crate::borrow_check::MirBorrowckCtxt<'cx, 'tcx> {
         old_load_end_span: Option<Span>,
     ) -> DiagnosticBuilder<'cx> {
         let via =
-            |msg: &str| if msg.is_empty() { msg.to_string() } else { format!(" (via `{}`)", msg) };
+            |msg: &str| if msg.is_empty() { msg.to_string() } else { format!(" (via {})", msg) };
```

Comment:
```suggestion
            |msg: &str| if msg.is_empty() { "".to_owned() } else { format!(" (via {})", msg) };
```

I know this wasn't you, but it seems less confusing to put it this way.